pkgs/by-name/yu/yunfaavatar/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, bash
+, makeWrapper
+, pciutils
+, x11Support ? true
+, ueberzug
+, fetchpatch
+, pkgs
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "yunfaavatar";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "yunfachi";
+    repo = "yunfaAvatar";
+    rev = version;
+    hash = "sha256-JbolO1I4K9+RBFJ/U611En4Rx5cG2+z1IcxYEeiaq6Q=";
+  };
+
+  outputs = [ "out" "man" ];
+
+  strictDeps = true;
+  buildInputs = [ bash pkgs.imagemagick ];
```

Comment:
```suggestion
  buildInputs = with pkgs; [ bash imagemagick ];
```

Reply:
I missed this entirely. These should be inputs.